### ISSUE TITLE
cdktf-cli diff output empty

### ISSUE THREAD

**Issue by MartinLeedotOrg** [hashicorp/terraform-cdk#2089]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

### cdktf & Language Versions
cdktf debug
language: go
cdktf-cli: 0.12.2
node: v14.18.1
cdktf: 0.12.2
constructs: 10.1.94
jsii: 1.66.0
terraform: 1.2.8
arch: x64
os: linux 4.18.0-348.20.1.el8_5.x86_64
go: go version go1.19 linux/amd64

### Expected Behavior

Plan with actions should produce output on `cdktf diff` or `cdktf apply`

### Actual Behavior

Output plan never appears.

I have tried with/without tmux, with multiple different terminal emulators and with FORCE_COLOR=0 set. Also with redirecting output to a file.

```
✗ cdktf diff
aws-ec2-dns  Creating Terraform Cloud configuration version
aws-ec2-dns  Created Terraform Cloud configuration version cv-413pLi61FHyrh4bh
aws-ec2-dns  Zipping up the directory cdktf.out/stacks/aws-ec2-dns
aws-ec2-dns  Uploading the directory to Terraform Cloud
aws-ec2-dns  Uploaded the directory to Terraform Cloud
aws-ec2-dns  Waiting for configuration version to become ready...
aws-ec2-dns  Creating speculative Terraform Cloud run
aws-ec2-dns  Created speculative Terraform Cloud run: https://terraform..../runs/run-2kZe8unmbRFzSWiy
aws-ec2-dns  Speculative Terraform Cloud run done
aws-ec2-dns  Getting plan output

➜ 
```

If I check the plan in the web UI I can see there are 2 resources to be created

### Important Factoids

We use a self hosted terraform enterprise instance

**Comment by MartinLeedotOrg** [comment]
Could we show it in the CLI? It would be very useful in cdktf deploy. I understand an edge use case for production but it would be very useful for rapid development.

**Comment by digitalrager** [comment]
I'm using Terraform Cloud and am unable to fully deploy from the CLI when I'm a team member in TC that has write permission because of this issue wont allow the plan to be displayed in the CLI.  Using the terraform CLI seems to work ok.  It seems these should line up ideally.

**Comment by DanielMSchmidt** [comment]
Fixed in #1955, now it should behave exactly the same as with Terraform CLI (since we are using Terraform CLI under the hood)
